tests/admin_app_tests/endpoint_tests/images_test.go

```diff
@@ -181,7 +183,225 @@ func TestPostImageFailedToCreateDatabaseEntry(t *testing.T) {
 	r.ServeHTTP(w, req)
 
 	assert.Equal(t, 500, w.Code)
+}
+
+func TestGetImage(t *testing.T) {
+	database_mock := mocks.DatabaseMock{
+		AddImageHandler: func(id string, file_name string, alt_text string, ext string) error {
+			return nil
+		},
+
+		GetImageHandler: func(id string) (common.Image, error) {
+			return common.Image{
+				Uuid:    id,
+				Name:    "test",
+				AltText: "default",
+				Ext:     ".png",
+			}, nil
+		},
+	}
+
+	r := admin_app.SetupRoutes(app_settings, database_mock)
+	post_recorder := httptest.NewRecorder()
+
+	pr, pw := io.Pipe()
+	writer := multipart.NewWriter(pw)
+
+	go func() {
+		defer writer.Close()
+
+		part, err := createTestForm(writer, "file", "test.png", "image/png")
+		if err != nil {
+			t.Error(err)
+		}
+
+		img := createImage()
+		if err != nil {
+			t.Error(err)
+		}
 
+		err = png.Encode(part, img)
+		if err != nil {
+			t.Error(err)
+		}
+	}()
+
+	// TODO: We have to create the image first. Maybe there's a better way to do this?
+	req, _ := http.NewRequest("POST", "/images", pr)
```

Comment:
There is another way: we could mock the access to the image files. I'm thinking of doing something like a `MediaVault` struct, then that could have `AddImage(...)`, `GetImage(...)` etc... But this is fine for this PR. I'll create some tasks for this

Reply:
Should be moved to a separate task. For now, I wouldn't want to increase the size of this PR further more.

Reply:
Yes